jbpm-system-test/jbpm-clustered-ejb-timers-test/.gitignore

```diff
@@ -0,0 +1,26 @@
+# Compiled class file
```

Comment:
Just a technical note. I noticed this doesn't include ignores for IDE files, maybe it could be included here as well. Or maybe the files could be unified. But this is not a blocker.

Reply:
added